conan/internal/runner/podman.py

```diff
@@ -0,0 +1,276 @@
+                _podman_info('Creating the container')
+                self.container = self.podman_client.containers.run(
+                    self.image,
+                    ["/bin/bash", "-c", "while true; do sleep 30; done"],
+                    name=self.name,
+                    volumes=volumes,
+                    environment=environment,
+                    user=self.configfile.run.user,
+                    privileged=self.configfile.run.privileged,
+                    cap_add=self.configfile.run.cap_add,
+                    security_opt=self.configfile.run.security_opt,
+                    detach=True,
+                    auto_remove=False,
+                    networks=self.configfile.run.network)
+            _podman_info(f'Container {self.name} running')
+        except Exception as e:
+            raise ConanException(f'Impossible to run the container "{self.name}" with image "{self.image}"'
+                                 f'\n\n{str(e)}')
+        try:
+            self.init_container()
+            self.run_command(self.command)
+            self.update_local_cache()
+        except ConanException as e:
+            error = True
+            raise e
+        except RunnerException as e:
+            error = True
+            raise ConanException(f'"{e.command}" inside container fail'
+                                 f'\n\nLast command output: {str(e.stdout_log)}')
+        finally:
+            if self.container:
+                error_prefix = 'ERROR: ' if error else ''
+                _podman_info(f'{error_prefix}Stopping container', error)
+                self.container.stop()
+                if self.remove:
+                    _podman_info(f'{error_prefix}Removing container', error)
+                    self.container.remove()
+
+    def build_image(self):
+        dockerfile_file_path = self.dockerfile
+        if os.path.isdir(self.dockerfile):
+            for df in ['Containerfile', 'Dockerfile']:
+                dockerfile_file_path = os.path.join(self.dockerfile, df)
+                if os.path.exists(dockerfile_file_path): break
+        build_path = self.podman_build_context or os.path.dirname(dockerfile_file_path)
+        ConanOutput().highlight(f"Container recipe path: '{dockerfile_file_path}'")
+        ConanOutput().highlight(f"Container build context: '{build_path}'\n")
+        _, podman_build_logs = self.podman_client.images.build(
+            dockerfile=dockerfile_file_path,
+            path=build_path,
+            tag=self.image,
+            buildargs=self.configfile.build.build_args,
+            cache_from=self.configfile.build.cache_from,
+        )
+        for chunk in podman_build_logs:
+            for line in chunk.decode("utf-8").split('\r\n'):
+                if line:
+                    stream = json.loads(line).get('stream')
+                    if stream:
+                        ConanOutput().status(stream.strip())
```

Comment:
No streaming of the output here. The output of the `build` command is spit out all at once.